<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordUpdateWidgets</class>
 <widget class="QDialog" name="RecordUpdateWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改病例</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>12</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>userId：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userIdEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>12</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>16</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>48</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>454</x>
     <y>12</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>42</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ageEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>42</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>42</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gendleBox">
   <property name="geometry">
    <rect>
     <x>454</x>
     <y>42</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>72</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>联系方式：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contactEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>72</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>72</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>职业：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="professionEdit">
   <property name="geometry">
    <rect>
     <x>454</x>
     <y>72</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>100</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>住址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>563</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>563</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>563</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="cancleBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存修改</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>129</y>
     <width>629</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0" columnstretch="0,0">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>个人病史：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="familyHistoryEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>131</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>家族病史：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="personalHistoryEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>131</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>userIdEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>nameEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>ageEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>gendleBox</zorder>
  <zorder>label_5</zorder>
  <zorder>contactEdit</zorder>
  <zorder>label_6</zorder>
  <zorder>professionEdit</zorder>
  <zorder>cancleBtn</zorder>
  <zorder>updateBtn</zorder>
  <zorder>addressEdit</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
